rllib/algorithms/dqn/dqn.py

```diff
@@ -660,6 +661,11 @@ def _training_step_new_api_stack(self):
                         sample_episodes=True,
                     )
 
+                    replay_buffer_results = self.local_replay_buffer.get_metrics()
+                    self.metrics.merge_and_log_n_dicts(
```

Comment:
Yeah, I wonder why `log_dict` doesn't work here. It should be the better choice here b/c we don't have more than one buffer.

```
self.metrics.log_dict(
    replay_buffer_results,
    key=REPLAY_BUFFER_RESULTS,
)
```

Maybe b/c in `replay_buffer_results` there are already `Stats` objects with their individual settings? ...

Reply:
I need to check it.

Reply:
So, basically, the lifetime metrics are somehow wrongly accumulated and grow exponentially. They probably need to be reduced before given to the `log_dict` method.